Workbooks/WaasUpdateInsights/SafeguardHolds/SafeguardHolds.workbook

```diff
@@ -49,7 +49,7 @@
             "multiSelect": true,
             "quote": "",
             "delimiter": ",",
-            "query": "let _SafeguardHold = \"SafeguardHold\";\r\nlet _SnapshotTime = datetime(\"{_SnapshotTime}\");\r\n\r\nUCUpdateAlert\r\n| where TimeGenerated >= _SnapshotTime\r\n| where AlertSubtype == _SafeguardHold\r\n| extend SafeguardHoldIDs = parse_json(AlertData).safeguardHoldIDs\r\n| mv-expand SafeguardHoldID=SafeguardHoldIDs\r\n| summarize by SafeguardHoldID=iff(isempty(SafeguardHoldID), \"Unknown\", tostring(SafeguardHoldID))\r\n| order by SafeguardHoldID asc",
+            "query": "let _SafeguardHold = \"SafeguardHold\";\r\nlet _SnapshotTime = datetime(\"{_SnapshotTime}\");\r\n\r\nUCUpdateAlert\r\n| where TimeGenerated >= _SnapshotTime\r\n| where AlertSubtype == _SafeguardHold\r\n| where isnotempty(DeploymentId)\r\n| extend SafeguardHoldIDs = parse_json(AlertData).safeguardHoldIDs\r\n| mv-expand SafeguardHoldID=SafeguardHoldIDs\r\n| summarize by SafeguardHoldID=iff(isempty(SafeguardHoldID), \"Unknown\", tostring(SafeguardHoldID))\r\n| order by SafeguardHoldID asc",
```

Comment:
DeploymentId is not always present for Safeguards. Client side (Appraiser based) Safegaurds will not have a DeploymentId. Can you get sign off from Dhiren.

Reply:
if we are trying to show SH for all the devices in UC, then this condition needs to be removed, because this is limiting to the devices from DSS , we want to show all the devices with client side and service side both.